src/System.Collections.Specialized/ref/4.0.1/System.Collections.Specialized.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see them throughout the dev/api branch.  Those aren't necessary either?  Or there's something specific about this project that makes it unnecessary?